sdk/identity/identity/review/identity.api.md

```diff
@@ -321,6 +321,7 @@ export { TokenCredential }
 // @public
 export interface TokenCredentialOptions extends CommonClientOptions {
     allowMultiTenantAuthentication?: boolean;
+    allowPiiLogging?: boolean;
```

Comment:
The name of this property is still being discussed.

Reply:
ok ok thank you! I’ll move it to draft then

Reply:
If this name is not being decided, giving that we’re about to reach code complete this Friday, should we push this to the next milestone?

Reply:
I think PII is no longer used by any of our data standards. I believe "End User Identifiable Information" (EUII) is the new equivalent. 

It also seems to fall under "Personal Data" in other places: https://www.microsoft.com/en-us/trust-center/privacy/customer-data-definitions